client/src/components/map/layers/networks-layer/index.tsx

```diff
@@ -0,0 +1,131 @@
+import { useMemo } from 'react';
+
+import { GeoJSONSourceSpecification } from 'maplibre-gl';
+import { Source, Layer, SymbolLayer } from 'react-map-gl/maplibre';
+
+import { LayerProps } from '@/types/layers';
+
+import { useMapNetworks } from '@/hooks/networks';
+
+export type NetworksLayerProps = LayerProps & {
+  beforeId?: string;
+};
+
+const getClusterSource = (source: GeoJSONSourceSpecification): GeoJSONSourceSpecification => ({
+  type: 'geojson',
+  data: source.data,
+  cluster: true,
+  clusterMaxZoom: 14, // Max zoom to cluster points on
+  clusterRadius: 100, // Radius of each cluster when clustering points
+  clusterProperties: {
+    organizationsCount: ['+', ['get', 'organizationsCount']],
+    projectsCount: ['+', ['get', 'projectsCount']],
+  },
+});
+
+const sharedLayout: SymbolLayer['layout'] = {
+  'icon-image': 'rect',
+  'icon-allow-overlap': true,
+  'icon-ignore-placement': true,
+  'text-size': 12,
+  'text-anchor': 'top',
+  'text-allow-overlap': true,
+  'text-ignore-placement': true,
+};
+
+const organizationsStyle: { layout: SymbolLayer['layout']; paint: SymbolLayer['paint'] } = {
+  layout: {
+    ...sharedLayout,
+    'icon-size': ['step', ['get', 'organizationsCount'], 1, 1, 1.5, 10, 2, 50, 2.5, 100, 2.5],
+    'icon-offset': [10, 0],
+    'text-field': ['number-format', ['get', 'organizationsCount'], { locale: 'en' }],
+    'text-anchor': 'left',
+    'text-offset': [0.75, 0],
+  },
+  paint: {
+    'text-color': '#fff',
+    'icon-color': '#2487E3',
+  },
+};
+
+const projectsStyle: { layout: SymbolLayer['layout']; paint: SymbolLayer['paint'] } = {
+  layout: {
+    ...sharedLayout,
+    'icon-size': ['step', ['get', 'projectsCount'], 1, 1, 1.5, 10, 2, 50, 2.5, 100, 2.5],
+    'icon-offset': [-10, 0],
+    'text-anchor': 'right',
+    'text-field': ['number-format', ['get', 'projectsCount'], { locale: 'en' }],
+    'text-offset': [-0.75, 0],
+  },
+  paint: {
+    'text-color': '#fff',
+    'icon-color': '#EF6A4C',
+  },
+};
+
+const paintBorder: SymbolLayer['paint'] = {
+  'icon-halo-color': '#000',
+  'icon-halo-width': 8,
```

Comment:
The border colour is #3C4363. The width is also different from what's on the design i.e. it should be 2px.

Reply:
The width doesn't work the same with the icon-halo-color. Not sure if we can find a way for it to be exact. For me it looks like 2px. Is hard to measure in this case.

Reply:
It looks like 1-1.5px on my screen 🙁. We really need to have a visual difference between clusters and markers.

Have you checked with Miguel if he knows a way to solve this? Could we have some HTML markers instead? 

If no solution is possible, then we need to talk to Lidia.

![The marker on top of France has a 1px black border which is not very visible compared to the clusters around](https://github.com/Orcasa-Platform/orcasa/assets/6073968/394c7c41-044f-4bec-b738-81946922eb79)